politeiad/cache/cockroachdb/decred.go

```diff
@@ -209,32 +213,32 @@ func (d *decred) cmdGetNumComments(payload string) (string, error) {
 		return "", err
 	}
 
+	// Lookup number of comments for provided tokens
 	type Result struct {
 		Token  string
 		Counts int
 	}
-
-	results := make([]Result, 0, 1024)
-
+	results := make([]Result, 0, len(gnc.Tokens))
 	err = d.recordsdb.
 		Table("comments").
 		Select("count(*) as counts, token").
 		Group("token").
 		Where("token IN (?)", gnc.Tokens).
 		Find(&results).
 		Error
-
 	if err != nil {
 		return "", err
 	}
 
-	commentMap := make(map[string]int)
+	// Put results into a map
+	numComments := make(map[string]int) // [token]numComments
```

Comment:
Adding the count here will help quite a bit with memory allocations.

Reply:
Fixed.